### DISCUSSION TITLE
About Lecture3 Quize Quetion3

### DISCUSSION THREAD

**Discussion by emjay73** [seominjoon/kaist-ai605#68] [Q&A] [answered]
About Quiz question 3, I get the point that modern deep learning methods might want to increase the model size to prevent overfitting, but it's quite confusing that I could sometimes increase and sometimes decrease the model size to prevent overfitting, which is all the options I got.
So could you briefly recap the options that I can choose based on the size of the dataset? (As mentioned in today's lecture?)
ex) overfitting with small dataset -> reduce the model size

**Comment by seominjoon** [comment]
Sorry fo the delayed response, I will discuss this during class!

**Comment by seominjoon** [comment, marked as answer]
Quoting what I said in the lecture: if the dataset size is small, the model might overfit (i.e. one should consider regularization or decrease the model size). If the dataset size is sufficiently large, the model is not likely to overfit (i.e. one should increase the model size). The exact borderline between the "small" and "big" is task-specific and not always clear, though. 

An additional note is that, this phenomenon is often called "double descent", where the test accuracy starts to drop at some point as the model size increases:

<img width="466" alt="Screen Shot 2021-09-13 at 11 23 54 AM" src="https://user-images.githubusercontent.com/1738325/133015017-6ac2aa15-07f1-4bd7-ba36-152ad2c5f92f.png">

This diagram is from the following recent paper, which is a quite good read:

https://arxiv.org/abs/2109.02355

**Reply by emjay73** [reply]
Thank you, Professor!
